src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        final String dataFromFile = readFile(fromFileName);
+        final String report = createReport(dataFromFile);
+        writeToFile(toFileName, report);
+    }
+
+    public String readFile(String fromFile) {
+        final StringBuilder stringBuilder = new StringBuilder();
+        File file = new File(fromFile);
+        try {
```

Comment:
Use try-with-resources
```suggestion
        try(BufferedReader infp = /* code */) {
```